dbux-code/src/net/serverUtil.js

```diff
@@ -6,35 +6,54 @@ import { newLogger } from '@dbux/common/src/log/logger';
 const { log, debug, warn, error: logError } = newLogger('net/servers');
 
 
+/**
+ * @param {number} port 
+ * @return {Promise<httpServer>}
+ */
 export function makeHttpServer(port) {
   const httpServer = http.createServer();
   // const address = '0.0.0.0';
   const address = '';
+
+  let _resolve, _reject;
```

Comment:
no need to store `resolve` or `reject` as local variables.

see Process._promise for reference